### PR TITLE
Add section about productionizing

### PR DESCRIPTION
**Pull request by pamelafox**
## Purpose

This README adds a section with tips for putting this template into production, based on customer experiences.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +184,38 @@ Once in the web app:
...
+## Productionizing
+
+This sample is designed to be a starting point for your own production application,
+but you should do a thorough review of the security and performance before deploying
+to production. Here are some things to consider:
+
+* **OpenAI Capacity**: The default TPM (tokens per minute) is set to 30K. That is equivalent
+  to approximately 30 conversations per minute (assuming 1K per user message/response).
+  You can increase the capacity by changing the `chatGptDeploymentCapacity` and `embeddingDeploymentCapacity` parameters in `infra/main.bicep` to your account's maximum capacity.
+  You can also view the Quotas tab from Azure OpenAI studio to understand how much capacity you have.
```
Comment (charris-msft) [inline README.md:196]: "from" > "in"

File: README.md [hunk 2]
```diff
@@ -183,6 +184,38 @@ Once in the web app:
...
+* **Azure Storage**: The default storage account uses the `Standard_LRS` SKU.
+  We recommend using `Standard_ZRS` for production deployments,
+  which you can specify using the `sku` property in `infra/main.bicep`.
```
Comment (charris-msft) [inline README.md:199]: ...`sku` property under `module storage` in ...

File: README.md [hunk 3]
```diff
@@ -183,6 +184,38 @@ Once in the web app:
...
+This sample is designed to be a starting point for your own production application,
+but you should do a thorough review of the security and performance before deploying
+to production. Here are some things to consider:
+
+* **OpenAI Capacity**: The default TPM (tokens per minute) is set to 30K. That is equivalent
+  to approximately 30 conversations per minute (assuming 1K per user message/response).
+  You can increase the capacity by changing the `chatGptDeploymentCapacity` and `embeddingDeploymentCapacity` parameters in `infra/main.bicep` to your account's maximum capacity.
+  You can also view the Quotas tab from Azure OpenAI studio to understand how much capacity you have.
+* **Azure Storage**: The default storage account uses the `Standard_LRS` SKU.
+  We recommend using `Standard_ZRS` for production deployments,
```
Comment (charris-msft) [inline README.md:198]: To improve your resiliency we recommend...

File: README.md [hunk 4]
```diff
@@ -183,6 +184,38 @@ Once in the web app:
...
+  which you can specify using the `sku` property in `infra/main.bicep`.
+* **Azure Cognitive Search**: The default search service uses the `Standard` SKU
```
Comment (charris-msft) [inline README.md:200]: Can you include a brief statement on why they should do this?

File: README.md [hunk 5]
```diff
@@ -183,6 +184,38 @@ Once in the web app:
...
+  with the free semantic search option. You should either change `semanticSearch` to "standard"
+  or disable semantic search entirely in the approaches files. If you see errors about search service capacity being exceeded, you may find it helpful to increase the number of replicas by changing `replicaCount` in `infra/core/search/search-services.bicep`.
```
Comment (charris-msft) [inline README.md:202]: in the /app/backend/approaches files.

**Review by charris-msft — CHANGES_REQUESTED** [review]
I LOVE THIS!!!

File: README.md (hunk 1 above)
Comment (chuwik) [inline README.md:196]: Would be nice to use a link for Azure OpenAI studio

File: README.md (hunk 5 above)
Comment (chuwik) [reply README.md:202]: I think it's in `infra/core/search/search-services.bicep`, or manually scaling it from the Azure Portal.

**Review by chuwik — APPROVED** [review]
Left a couple of nits, otherwise looking good!

**Review by charris-msft — APPROVED** [review]

**Pull request merged**